seed/challenges/03-front-end-libraries/front-end-libraries-projects.json

```diff
@@ -96,8 +96,10 @@
       "description": [
         "Fulfill the user stories by getting all of the tests to pass. Use whichever libraries you need. Give it your own personal style.",
         "Here's a <a href='http://codepen.io/freeCodeCamp/full/wgGVVX' target='_blank'>working example</a>. Try not to look at its code.",
-        "You can build your project by forking <a href='http://codepen.io/FCC_test_suites/pen/JbgXab' target='_blank'>this CodePen pen</a>. Or you can use this CDN link to run the tests in any environment you like: <code>https://gitcdn.link/repo/no-stack-dub-sack/testable-projects-fcc/master/build/bundle.js</code>",
-        "Once you're done, submit below the URL to your working project with all its tests passing.",
+        "<strong>Note On Calculator Logic:</strong> It should be noted that there are two main schools of thought on calculator input logic: <dfn>immediate execution logic</dfn> and <dfn>formula logic</dfn>. Our example utilizes formula logic and observes order of operation precedence, immediate execution does not. Either is acceptable, but please note that depending on which you choose, your calculator may yield different results than ours for certain equations (see below example). As long as your math can be verified by another production calculator, please do not consider this a bug. <strong>EXAMPLE:</strong>",
```

Comment:
Break this into multiple lines:

![image](https://cloud.githubusercontent.com/assets/1884376/22309986/9995faaa-e373-11e6-94bb-ba2d376d3db4.png)